packages/devtools_app/lib/src/shared/console/console_service.dart

```diff
@@ -164,11 +164,7 @@ class ConsoleService extends Disposer {
   ///
   /// Note that this output might be truncated after significant output.
   ValueListenable<List<ConsoleLine>> get stdio {
-    assert(
-      _serviceInitialized,
-      '`ConsoleService.ensureServiceInitialized` must be called before '
-      'interacting with the ConsoleService.',
-    );
+    ensureServiceInitialized();
```

Comment:
This is probably ok but not ideal. It will mean the the console is guaranteed to show up empty the first frame it is displayed but does ensure we don't subscribe unless we need to. @kenzieschmoll any thoughts on this judgement call?